consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java

```diff
@@ -192,7 +213,11 @@ private boolean processMessage(final SignedData<? extends Payload> msg, final Me
     } else if (isMsgForFutureChainHeight(msgRoundIdentifier)) {
       addMessageToFutureMessageBuffer(msgRoundIdentifier.getSequenceNumber(), rawMsg);
     } else {
-      LOG.info("IBFT message discarded as it is from a previous block height");
+      LOG.info(
```

Comment:
trace (maybe debug?)

Reply:
the other discard logs are at debug. so thinking debug for this one too.